docs/kf.dev/content/en/docs/getting-started/install/_index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Install Kf"
+linkTitle: "Basic Kf Installation"
+weight: 10
+description: >
+  Learn how to use the `kf` CLI to create a new Kubernetes cluster with Kf
+  installed.
+---
+
+[for-operators]: /docs/operators
+Follow these instructions to quickly configure and deploy a basic Kf
+installation on a new Google Kubernetes Engine (GKE) cluster. For a more
+comprehensive installation guide for system operators/admins, please see the
+[Operators Guide][for-operators].
+
+## Prerequisites
+
+### Google Cloud
+[gke]: https://cloud.google.com/kubernetes-engine/
+[free]: https://cloud.google.com/free
+[new-project]: https://cloud.google.com/resource-manager/docs/creating-managing-projects
+These instructions assume Kf will be installed onto [Google Kubernetes Engine
+(GKE)][gke] in a Google Cloud Platform (GCP) project. If you do not have a GCP
+project, you can sign up for the [free tier][free].
```

Comment:
I'm not sure if our installation works within the free tier, especially with #486 unresolved.

Reply:
It doesn't work in the free tier, but the $300 credit makes it do-able. I'm going to change the language to focus on sign up, and mention a user may be eligible for the credit.